ot-node.js

```diff
@@ -128,13 +123,13 @@ class OTNode {
             this.logger.info('Operational database module: sequelize implementation');
             // eslint-disable-next-line global-require
             const db = require('./models');
-            // todo change if statement to if (this.config.otNodeUpdated);
-            if (process.env.NODE_ENV !== 'test') {
+            
+            if(this.config.otNodeUpdated) {
                 execSync('npx sequelize --config=./config/sequelizeConfig.js db:migrate');
-                // todo remove UPDATE file for next release
-                // execSync('rm ${updateFilePath}');
-                // this.config.otNodeUpdated = false;
+                execSync(`rm ${updateFilePath}`);
```

Comment:
Please use fileService (this will be a module once we refactor it) and call deleteFile method